docs/Schema-changes.md

```diff
@@ -0,0 +1,164 @@
+        "entity_status_effective"
+      ]
+    },
+    "address_fields": [],
+    "search_fields": [
+      "registration_id",
+      "entity_name"
+    ],
+    "labels": {
+      "schema": {
+        "en": "Registration Credential"
+      },
+      "schema_description":{
+      	"en": "Registration Description"
+      },
+      "attributes": [
+        {
+          "name": "registration_id",
+          "translations": {
+            "en": {
+              "label": "Registration ID",
+              "description": "Registration/Incorporation Number or Identifying Number"
+            }
+          }
+        },
+		
+        ...
+		
+        {
+          "name": "reason_description",
+          "translations": {
+            "en": {
+              "label": "DEMO",
+              "description": "Reason for credential issue/update"
+            }
+          }
+        },
+        {
+          "name": "expiry_date",
+          "translations": {
+            "en": {
+              "label": "Credential Expiry Date",
+              "description": "Date Credential expired"
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+~~~
+*Figure 4: Example registration schema*
+
+##### Setting the schema label:
+In order to set the schema label add the following JSON object to `metadata->labels`
+~~~
+"schema": {
+	"en": "<YOUR SCHEMA NAME, Example: Registration >"
+},
+"schema_description":{
+	"en": "<YOUR SCHEMA DESCRIPTION, Example: Credential proving an entity's registration>"
+},
+~~~
+
+
+##### Setting the credential title and highlighted attributes:
+To set the credential title and highlighted attributes add the following JSON object to `metadata`:
+~~~
+"highlighted_attributes":[
+	"<YOUR HIGHLIGHTED ATTRIBUTE KEY 1>",
+	"<YOUR HIGHLIGHTED ATTRIBUTE KEY 2>"
+  ],
+"credential_title":"<YOUR CREDENTIAL TITLE KEY>",
+~~~
+highlighted attribute keys and the credential title key refer to the attribute names on each of the attributes in the schema. An example is shown in figure 4.1. Highlighted attributes will appear on credential cards in the order they're listed on the schema. Example: attribute #1 will appear above attribute #2 on this credential card.
+
+![](./assets/attribute_key.png)
+*Figure 4.1: Example attribute key*
+
+##### Setting the credential title and highlighted attribute labels:
+Attribute labels appear to the left of an attribute value on a credential card. The attribute label describes the attribute value being displayed on the credential card. See figure 4.2 for an example.
+
+![](./assets/attribute_label_value.png)
+*Figure 4.2: Attribute label and value*
+
+To set the attribute label simply edit the claim label for the attribute in question. To do so, in `metadata->labels->attributes->translations->[locale]->label` change the `name` field to your desired label as seen in figure 4.3 where we change the `reason_description` from `Reason` to `DEMO`. See figure 4.4 for the resulting change in Orgbook
```

Comment:
Consider using the more standard `.` notation for describing the path in the json (throughout the document).  For example:
- `metadata->labels->attributes->translations->[locale]->label` should be `metadata.labels.attributes.translations.[locale].label`